packages/hash/api/src/graphql/resolvers/entity/aggregateEntity.ts

```diff
@@ -13,8 +13,8 @@ export const dbAggregateEntity =
   async (params: { accountId: string; operation: AggregateOperationInput }) => {
     const { accountId, operation } = params;
     const { entityTypeId } = operation;
-    const pageNumber = operation?.pageNumber || 1;
-    const itemsPerPage = operation?.itemsPerPage || 10;
+    const pageNumber = operation?.pageNumber ?? 1;
+    const itemsPerPage = operation?.itemsPerPage ?? 10;
```

Comment:
I don't think we should change these as `0` for either is an invalid input. Probably we should have a `@todo` to throw an error if someone provides `0`.

Reply:
Reverted this